starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,46 @@
 // Names and Input
 
+let hacker1 = 'Kiryl';
+let hacker2 = 'mike';
+console.log( `The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
 
 //Conditionals
 
+if(hacker1.length > hacker2.length) {
+    console.log(`The Driver has the longest name, it has ${hacker1.length} characters`);
+} else if(hacker2.length > hacker1.length) {
+    console.log(`Yo, navigator got the longest name, it has ${hacker2.length} characters`);
+} else {
+    console.log(`wow, you both got equally long names, ${hacker1.length} characters!!`);
+}
 
-// Lorem ipsum generator
+let arrName = hacker1.split('');
+let result = [];
+for(let i = 0; i < arrName.length; i++) {
+  result.push(arrName[i].toUpperCase());
+}
+console.log(result.join(' '));
+
+console.log(hacker2.split('').reverse().join(''));
+
+if( hacker1 > hacker2) {
```

Comment:
In order to check which name goes first in alphabetical order, you can use 
```.localeCompare()```. Let me know if you need help.